SamsungPay/src/main/java/com/braintreepayments/api/SamsungPayStartListener.java

```diff
@@ -0,0 +1,34 @@
+package com.braintreepayments.api;
+
+import androidx.annotation.NonNull;
+
+import com.samsung.android.sdk.samsungpay.v2.payment.CardInfo;
+import com.samsung.android.sdk.samsungpay.v2.payment.CustomSheetPaymentInfo;
+import com.samsung.android.sdk.samsungpay.v2.payment.sheet.CustomSheet;
+
+/**
+ * Callback for receiving results of
+ * {@link SamsungPayClient#startSamsungPay(CustomSheetPaymentInfo, SamsungPayStartListener)}.
+ */
+public interface SamsungPayStartListener {
+
+    /**
+     * Called when Samsung Pay card info has been updated. {@link SamsungPayClient#updateCustomSheet(CustomSheet)}
+     * @param cardInfo SamsungPay [CardInfo]
+     * @param customSheet SamsungPay [CustomSheet]
+     */
+    void onSamsungPayCardInfoUpdated(CardInfo cardInfo, CustomSheet customSheet);
```

Comment:
Are these @NonNull?

Reply:
Good call, will have to check.

Reply:
I think they're all non-null here since it's more listener style than callback / result-error style. I updated the annotations in the listener.